src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -69,6 +71,10 @@ public class BigFormat extends AbstractSSTableFormat<BigTableReader, BigTableWri
 {
     private final static Logger logger = LoggerFactory.getLogger(BigFormat.class);
 
+    // Feature flag env variable to use extended TTL up to 2106 with the new oa sstable format. See c14227
```

Comment:
I'd add a brief description of the purpose of each enum value.

Reply:
Done